.github/workflows/cargo-lints.yml

```diff
@@ -50,4 +50,5 @@ jobs:
 
       - name: Check
         run: |
-          cargo clippy --all
+          cargo clippy
+          cargo clippy -p node
```

Comment:
with `wasmer/js`, it was not possible to run build / check against non-wasm target to `crates/wasm` anymore - specifically in this case wasmer doesn't bring its transitive deps for non-working target (https://github.com/wasmerio/wasmer/blob/master/lib/api/Cargo.toml#L58-L62). Not sure if there's way to override, but also it feels unreasonably complex work to bring few workflows.

Reply:
Just read this. Happy to accept contributions to improve this! :)